docs/What_Is_A_Custom_Device.md

```diff
@@ -0,0 +1,172 @@
+```
+
+&nbsp;&nbsp;&nbsp; **Note:** &lt;xxxx&gt; is the  VeriStand version number.
+
+VeriStand parses **Common Data\Custom Devices** for custom device XML files when it first launches. You must restart VeriStand to recognize newly added or modified custom device XML files. 
+
+Add the custom device to the system definition in the configuration tree by navigating to **[System Definition](https://zone.ni.com/reference/en-XX/help/372846M-01/veristand/system_explorer/)** » **[Targets](http://zone.ni.com/reference/en-XX/help/372846B-01/veristand/targets_se/)** » **[Controller](https://zone.ni.com/reference/en-XX/help/372846M-01/veristand/cp_controller/)** and right-clicking **[Custom Devices](https://www.ni.com/documentation/en/veristand/latest/manual/add-configure-custom-device/)**. The following image displays how-to add a custom device.
+
+![](images/Picture4.jpg)
+
+Logically, custom devices consist of three parts.
+
+&nbsp;&nbsp;&nbsp; Custom Device Framework<br />
+&nbsp;&nbsp;&nbsp; Custom Code<br />
+&nbsp;&nbsp;&nbsp; Custom Device XML File<br />
+
+#### Custom Device Framework
+
+The custom device framework consists of type definitions, specifically named controls and indicators, template VIs and a LabVIEW API. Together these items for the rules, or framework, that allows any conforming VI to interact with VeriStand. There are several prebuilt types of custom devices. Almost any requirement can be accomplished by adding or modifying code in one of the prebuilt devices.
+
+The prebuilt devices start with the Custom Device Template Tool. The template tool is located in <span style="color:green">*<vi.lib>\NI VeriStand\Custom Device Tools\Custom Device Template Tool\Custom Device Template Tool.vi*</span>.
+The developer specifies the type of custom device before running the template tool. The tool generates the LabVIEW Project for the new custom device. The exact resources in the project depend on the type of custom device selected.
+
+The project is pre-populated with VIs, LabVIEW Libraries, an XML File, and two build specifications. These resources provide the framework upon which almost all custom devices are built.
+
+VeriStand evolved from NI Dynamic Test Software (NI-DTS). NI-DTS evolved from 3rd party intellectual property (IP) called EASE. The IP made basic provisions for add-on LabVIEW code. This provisions could be considered the first custom device framework on which several “custom devices” were built. If you find a custom device that does not fit the Custom Device Template Tool framework, you may be operating an EASE based custom devices.
+
+For each of the five types of custom devices, you’ll see two VI libraries in the LabVIEW source project: <span style="color:green">*Custom Device API.lvlib*</span> and <span style="color:green">*Custom Device Name Custom Device.lvlib.*</span>
+
+The following image displays a new custom device project that uses these two libraries.
+![](images/Picture5.jpg)
```

Comment:
We need to replace this image, with:
![image](https://user-images.githubusercontent.com/62251693/131471135-a5f78979-34a0-403d-b89f-e719d485c506.png)